src/router/Router.jsx

```diff
@@ -0,0 +1,26 @@
+import { Route, Routes } from "react-router-dom";
+import { Suspense, lazy } from "react";
+
+const LandingPage = lazy(() => import("../pages/landing-page/LandingPage"));
+const ApiResponse = lazy(() => import("../pages/api-response/ApiResponse"));
+const Explore = lazy(() => import("../pages/explore/Explore"));
+const History = lazy(() => import("../pages/history/History"));
+const Playlist = lazy(() => import("../pages/playlist/Playlist"));
+const WatchLater = lazy(() => import("../pages/watch-later/WatchLater"));
+const Liked = lazy(() => import("../pages/liked-videos/LikedVideos"));
```

Comment:
I like how lazy loading is handled here. I am also going to do this.

Reply:
There are couple of things that you need to take care of. You need to export the component as default to lazy load it.